# Introduction To Cancer Genomic Analysis with Next Generation Sequencing (NGS)
### Requirement: 
#### - Students with basic biology and molecular biology
#### - Laptop (Storage 10GB available and 8GB RAM) with either Windows/MacOS/Ubuntu Operating System
#### - Good internet wifi available
#
## 1. Introduction to Next Generation Sequencing (NGS), Human Genome and related Databases
### 1.1. [UCSC Genome Browser](https://genome.ucsc.edu/)
### 1.2. [Gencode](https://www.gencodegenes.org/human/)
##
## 2. Operating System and Programming Language
### 2.1. Linux Operating System
### 2.2. Linux Commands
### 2.3. Bash Script
### 2.4. Google Colab
##
## 3. DNA amplification for enrichment
### 3.1. Nucleotide and Protein Database
### 3.2. [BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi)
### 3.3. [PSI-BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch&PROGRAM=blastp&BLAST_PROGRAMS=psiBlast)
### 3.4. PCR and Primer/Probe Design
##
## 4. Introduction to Cancer Genomics
### 4.1. Introduction to The Cancer Genome Atlas Database (TCGA) via [cBioPortal](https://www.cbioportal.org/)
### 4.2. Library Preparation for Target Sequencing
##
## 5. Upstream Analysis for Target Sequencing to detect germline variant
### 5.1. Raw Data processing (Quality control, trimming & filtering)
### 5.2. Sequence Alignment (Mapping)
##
## 6. Downstream Analysis for Target Sequencing to detect germline variant
### 6.1. Variant calling and Filtering
### 6.2. Variant annotation
##
## 7. Variant Interpretation and Genetic counseling
### 7.1. Double-check pathogenic variants with Sanger sequencing
### 7.2. Clinvar Database and Variant Interpretation following ACMG 2015
### 7.3. Genetic counseling
##
## 8. Discussion and Summary: Somatic Variant detection using 
### 8.1. Tissue/FFPE 
### 8.2. Circular tumor DNA (ctDNA)
##

